### PR TITLE
Ecom pages - Landing Page

### PR DESCRIPTION
**Pull request by rkapoor10**
landing

### CODE REVIEW

File: landing/landing.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    height: 15rem;
+}
+
+
+.content, .background{
+    grid-column: 1/2;
+    grid-row: 1/2;
+}
+
+.category-wrapper, .brands-wrapper{
```
Comment (Priya-Patel31) [inline landing/landing.css]: give gap as 1rem

File: index.html [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+                            <div class="title center white content">
+                                <h3 class="fs-2">Racquets</h3>
+                                <p class="txt-s ">Starting at Rs. 1245</p>
+                            </div>
+
+                        </div>
+
+                    </a>
+                </div>
+                <div class="catg-card">
```
Comment (Priya-Patel31) [inline index.html:72]: give display: flex and align-items :center

File: landing/navbar.css [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+    align-items: center;
+    justify-content: space-between  ;
+    border-bottom: 1px solid #494b4c;
+    padding: 5rem 0;
+}
+.footer-middle-wrapper{
+    display: flex;
+    align-items: center;
+    justify-content: space-between  ;
+    border-bottom: 1px solid #494b4c;
```
Comment (Priya-Patel31) [inline landing/navbar.css:129]: use variable for these colors!!

File: ecstylesheets/base.css [hunk 4]
```diff
@@ -0,0 +1,30 @@
+:root{
+    --baseshade-gray: #434343;
```
Comment (JantuDeb) [inline ecstylesheets/base.css:2]: rallyalley-logo-removebg-preview

File: landing/navbar.css [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+    
+}
+
+/* .sapce-evenly{
+    display: flex;
+    justify-content: space-evenly;
+} */
+
+.footer-container-ec{
+    margin-left: auto;
```
Comment (Priya-Patel31) [inline landing/navbar.css:112]: just a suggestion that you can use shorthand property of margin as margin : 0 auto ;

**Review by Priya-Patel31 — CHANGES_REQUESTED** [review]
do make these changes as they are necessary !!

File: landing/landing.css [hunk 6]
```diff
@@ -0,0 +1,97 @@
+
+.landing-page-layout{
+    display: grid;
+    grid-template-areas: 
+    "head head head head head head"
+    "bann bann bann bann bann bann"
+    "catg catg catg catg catg catg"
+    "prod prod prod prod prod prod"
+    "brnd brnd brnd brnd brnd brnd";
```
Comment (JantuDeb) [inline landing/landing.css:9]: Use a better name

**Review by JantuDeb — APPROVED** [review]
Everything looks neat and clean.

**Review by JantuDeb — APPROVED** [review]
You can merge it now

File: landing/landing.css (hunk 1 above)
Comment (rkapoor10) [reply landing/landing.css]: done

File: index.html (hunk 2 above)
Comment (rkapoor10) [reply index.html:72]: done

File: landing/navbar.css (hunk 5 above)
Comment (rkapoor10) [reply landing/navbar.css:112]: ok

**Review by Priya-Patel31 — APPROVED** [review]
Changes have been made now you can merge it !!

**Pull request merged**